Gemfile

```diff
@@ -72,6 +72,7 @@ gem 'aws-sdk-core', '~> 2.2.15'
 # Optional Services.
 gem 'omniauth-37signals'          # BasecampAgent
 gem 'omniauth-wunderlist', github: 'wunderlist/omniauth-wunderlist', ref: 'd0910d0396107b9302aa1bc50e74bb140990ccb8'
+gem 'omniauth-lifx'
```

Comment:
I feel like then we're forcing people to modify the Gemfile which is checked into git, and would potentially cause merge conflicts as they try to stay up to date. I currently feel a lot of pain around app start up time, but haven't felt any pain installing a lot of gems.